any.hpp

```diff
@@ -237,14 +246,30 @@ static_any<_N>::static_any(static_any<_M>&& another)
 template <std::size_t _N>
 template <class _T, class>
 static_any<_N>& static_any<_N>::operator=(_T&& t)
+{
+	constexpr const bool canThrow =
+			(std::is_rvalue_reference<_T&&>::value && std::is_nothrow_move_constructible<_T>::value)
+			|| (std::is_lvalue_reference<_T&&>::value && std::is_nothrow_copy_constructible<_T>::value);
```

Comment:
Now I review my own code I see the condition should be the other way around - I forgot the negation in fromt of the `is_nothrow..`. I will change that.